integration/benchmarks/rax/run_benchmark.py

```
@@ -0,0 +1,656 @@
+    # Use a single clean representative search query command per setup for memtier stability
+    q_rep = active_queries[0] if active_queries else "term"
+    memtier_cmd.append(f'--command=FT.SEARCH bench_idx {q_rep} LIMIT 0 10')
+
+
+
+
+
+        
+    print(f"Executing memtier_benchmark ({MEMTIER_THREADS} threads, {MEMTIER_CLIENTS} client/thread, target={TOTAL_SEARCH_QUERIES} queries)...")
+    search_start = time.perf_counter()
+    subprocess.run(memtier_cmd, check=True, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)